### PR TITLE
CIV-11181 CCJ Navigation

### PR DESCRIPTION
**Pull request by portizdev**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-11181

### Change description ###
Fix for redirection bug in CCJ flows by creating a new controller to differentiate flow 'county-court-judgement' and 'ccj' in paid amount summary screen

- county-court-judgement : redirects to claimant response task-list
- ccj: redirects to payment options

Fix tests

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by neeta-hmcts — APPROVED** [review]

File: src/main/routes/features/claimantResponse/ccj/judgmentAmountSummaryExtendedController.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    claimFee,
+    judgmentSummaryDetails,
+  });
+}
+
+judgmentAmountSummaryExtendedController.get(CCJ_EXTENDED_PAID_AMOUNT_SUMMARY_URL, async (req: AppRequest, res: Response, next: NextFunction) => {
+  try {
+    const lang = req.query.lang ? req.query.lang : req.cookies.lang;
+    const claim = await getCaseDataFromStore(req.params.id);
+    const claimFee = await civilServiceClient.getClaimAmountFee(claim?.totalClaimAmount, req);
```
Comment (laarmada) [inline src/main/routes/features/claimantResponse/ccj/judgmentAmountSummaryExtendedController.ts:32]: this ? in claim?. is not necessary we always return a claim

**Review by laarmada — APPROVED** [review]
minor comment

**Review by kenneth-hmcts — DISMISSED** [review]

**Review by laarmada — CHANGES_REQUESTED** [review]
you have a important sonar problems:
https://sonarcloud.io/summary/new_code?id=civil-citizen-ui&pullRequest=2385

File: src/main/routes/features/claimantResponse/ccj/judgmentAmountSummaryExtendedController.ts (hunk 1 above)
Comment (portizdev) [reply src/main/routes/features/claimantResponse/ccj/judgmentAmountSummaryExtendedController.ts]: Done

**Review by laarmada — APPROVED** [review]

**Review by kenneth-hmcts — APPROVED** [review]

**Review by ElifDenizEsea — APPROVED** [review]

**Pull request merged**